themes/Makefile

```diff
@@ -0,0 +1,16 @@
+BASE_THEME	= yeti
+THEMES_DIR	= themes
+OUTPUT_DIR	= ../inginious/frontend/static/css/themes
+
+compile:
+	if [ ! -d "node_modules" ]; then make install; fi
+	if [ ! -d "$(OUTPUT_DIR)/$(theme)" ]; then cp -r $(OUTPUT_DIR)/default $(OUTPUT_DIR)/$(theme); fi
+	./node_modules/grunt/bin/grunt swatch:$(theme)
+	mv $(THEMES_DIR)/$(theme)/bootstrap.min.css $(OUTPUT_DIR)/$(theme)/.
```

Comment:
There should clearly be way to modify `Gruntfile.js` in order to compile all themes as the default action, and to copy the `bootstrap.min.css` in the appropriate folder.

This way, this whole Makefile would be kind of useless and compiling the themes would just consists in `npm install` and `grunt`.